Ghidra/Framework/Project/src/main/java/ghidra/framework/model/ProjectData.java

```diff
@@ -88,12 +88,11 @@
 	public DomainFile getFileByID(String fileID);
 
 	/**
-	 * Get a GhidraURL for a shared domain file which is available 
+	 * Get a URL for a shared domain file which is available 
 	 * within a remote repository.
 	 * @param path the absolute path of domain file relative to the root folder.
 	 * @return URL object for accessing shared file from outside of a project, or
 	 * null if file does not exist or is not shared.
-	 * @see ghidra.framework.data.URLGhidraRepository
 	 */
 	public URL getSharedFileURL(String path);
```

Comment:
This was a bit confusing. It appears that a standard URL is being returned so I changed the above to URL instead of GhidraURL. Also ghidra.framework.data.URLGhidraRepository no longer exists.